content/en/docs/examples/alb-nlb/index.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Securely direct traffic from IBM Cloud Kubernetes Service Ingress to Istio Ingress Gateway
```

Comment:
This title is a bit too long, especially if you view it from preview, it is shown on the left panel.

I think it would be good to have 2 blogs.  I found it confusing that NLB and ALB and talked together while users configure NLB to istio ingress gateway and/and ALB to istio ingress gateway.  thoughts?

Reply:
@rvennam ^ what do you think?